xExtension-YouTube/configure.phtml

```diff
@@ -1,38 +1,41 @@
-
-<form action="<?php echo _url('extension', 'configure', 'e', urlencode($this->getName())); ?>" method="post">
-	<input type="hidden" name="_csrf" value="<?php echo FreshRSS_Auth::csrfToken(); ?>" />
+<form action="<?= _url('extension', 'configure', 'e', urlencode($this->getName())) ?>" method="post">
+	<input type="hidden" name="_csrf" value="<?= FreshRSS_Auth::csrfToken() ?>" />
 	<div class="form-group">
 
-		<label class="group-name" for="yt_height"><?php echo _t('ext.yt_videos.height'); ?></label>
+		<label class="group-name" for="yt_height"><?= _t('ext.yt_videos.height') ?></label>
```

Comment:
Event though short echo tags are officially first class citizens in PHP since 5.4 (and we can safely argue that we give a shit on users with 5.3), I don't like them ([no matter what PSR-1 says](https://www.php-fig.org/psr/psr-1/#21-php-tags)) - but even more important: a PR should not change formattings or method names or code styles without good reasons. It makes the life of the reviewer so much more complicated.
As `<?php` vs `<?=` is a semi-religious discussion (like tabs vs spaces) please revert them.

Reply:
Just a note to say that FRSS now supports only from PHP 5.6 and above, and use short tags to reduce the amount of `echo` in the code since the last version. That being said, it's your extension so your choices of course :)

Reply:
Thanks for the feedback! I haven't worked on a project that is using PHP templates for a long time, so maybe this is more state of the art nowadays. Maybe it is just me, using PHP for almost 2 decades now and having a lot of troubles with them in the past ... old habits are very strong. But as I said "semi-religious discussion" and those are vey rarely driven by logic ^^ So this is one of the points where I will shamelessly use my repository owner decision power 😈

Reply:
Done.